scss/standalone/patterns_equal-height-row.scss

```diff
@@ -4,7 +4,10 @@
 // dependencies needed for examples
 @include vf-p-buttons;
 @include vf-p-media-container;
+@include vf-p-image;
 @include vf-p-headings;
```

Comment:
Looks like standalone is missing the import for muted headings
```suggestion
@include vf-p-headings;
@include vf-p-muted-heading;
```
Non-standalone vs standalone:
<img width="354" alt="Screenshot 2024-08-22 at 5 29 00 PM" src="https://github.com/user-attachments/assets/3dec422a-9544-4540-b3b4-7dcb459f55a5">
<img width="354" alt="Screenshot 2024-08-22 at 5 29 03 PM" src="https://github.com/user-attachments/assets/17dfe4cb-f78c-4634-b686-deb3db4d387f">